pages/Woodcutter-Androids.md

```diff
@@ -0,0 +1,15 @@
+A specialized android is created to automate the process of harvesting wood and replanting it.
+This android will work with all types of wood.
+
+## Obtaining
+Woodcutter Androids can be crafted in an [Enhanced Crafting Table](https://github.com/Slimefun/Slimefun4/wiki/Enhanced-Crafting-Table).
+
+## Usage
+This android will cut and replant any type of wood they're facing.
+
+## Tiers
+There is only one tier in this woodcutter android.
+
+| Tier | Fuel Efficiency |
+| ---- | --------------- |
+| Basic | 1.0x |
```

Comment:
```suggestion
| Tier  | Fuel Efficiency |
| ----- | :-------------: |
| Basic | 1.0x            |
```

Reply:
What is the difference on this one?

Reply:
nvm found the ultimate markdown cheatsheet.